### ISSUE TITLE
Supervisor Meeting - 09/25/2025

### ISSUE THREAD

**Issue by rawanmahdi** [speech-buddies/VoiceBridge#14]
**Date & Time**
September 25th, 2025, 11:00 - 11:30AM

**Attendance**

- [X] Kelvin
- [X] Luna
- [X] Mazen
- [X] Rawan

**Agenda**

- share repository
    - mention issues are primary mode of creating tasks and documenting ideas
- share the documents we have completed thus far (problem statement/goals/dev plan)
- discuss POC plan in depth
- 

**Questions to Ask**

-

**Comment by luna-aljammal** [comment]
Meeting Minutes:
- Showed Dr.Brodbeck issues and shared that he can use this feature
- Gave a quick preview of our project repository + documents so far (Problem Statement, Goals and Development Plan (Focused on PoC))
- Dr.Brodbeck recommended making latency an expected goal rather than a stretch goal
    - made sense to the team, will measure by a time range or threshold (e.g., <1s delay)
- Conduct a systematic, literary review before starting technical implementation
	- Create a table with existing models for typical and dysarthric speech (use research papers, etc.), could be used in our 'extra' as a report
	- Include key features, references/existing uses of the models, latency, source code-ability, note whether it has already been adapted for dysarthric speech or not
- When evaluating existing models, consider which can be run locally vs require cloud resources
	- Consider the advantage of running it locally (Improves privacy, minimizes costs)
- Could reach out if compute resources are a blocker (Compute Canada resources)
- To perform **data augmentation**, can add background noise in varying levels to the datasets (part of stretch goals)
- Should investigate if the features are additive, wrt synthetic data generation, attempting to manipulate normal speech into dysarthric speech might require more effort than we expected (team considered exploring it quickly, but Dr.Brodbeck suggested it'd be complex and time consuming) due to synthetic data being more homogenous in nature
- We'll explore measuring subjective intelligibility (consider objective intelligibility?)

Suggestions for PoC:
- Explore filtering phonemes with motor actions (when pronouncing 't', the tongue touches the palette), requires research
- When sound (phonemes) is trained for graphemes (letters), through analysis of a typical transcription, can find loose connections between the otology and phonology
- As an extra, could explore how different disorders can affect different phonemes based on which motor aspects are impaired
	- Can map the impacted motor feature with the intended motor feature
- Consider decomposing the functions
	- Could pretrain the model first to detect which phonemes were impacted, predict which motor features are responsible (based on the speech variations), then transcribe
- End-to-end project could be the acoustics to transcripts, could include predicting the next datapoints as part of it
- Interesting publications to look at from Carol Epsy-Wilson (Modeling of Articulatory Gestures to Control Effects of Production Variability on Speech Technologies, Hybrid Convolutional Neural Networks For Articulatory And Acoustic Information Based Speech Recognition, Recognizing articulatory gestures from speech for robust speech recognition https://ece.umd.edu/clark/faculty/391/Carol-Espy-Wilson

**Comment by rawanmahdi** [comment]
Rersearch Question:

1. which models are most suitable for our use case? Specifically for irregular speech.
2. fine tuning pre-trained on impaired speech? see whats already been done and what has failed. 
3. data pre-processing - denoising, amplifying the desired signals, etc for irregular speech. 
4. feature engineering - motor features/phonemes as a feature.
